ui/app/src/matrix-store.ts

```diff
@@ -1513,6 +1552,32 @@ export class MatrixStore {
 
     const hcPort = import.meta.env.VITE_AGENT === "2" ? import.meta.env.VITE_HC_PORT_2 : import.meta.env.VITE_HC_PORT;
     const appletAppAgentWebsocket = await AppAgentWebsocket.connect(`ws://localhost:${hcPort}`, appInfo.installed_app_id);
+
+      // const devhubHappReleaseHash =
+      //   this.releaseHashOfAppletInstance(appletInstanceId)!;
+      // ATTENTION: IT IS ASSUMED HERE THAT THE APPLET IS ALREADY IN THE MATRIX!!
```

Comment:
TODO: refactor/abstract for duplication I guess? Appreciate this is a WIP though and that Duplication is Far Cheaper Than the Wrong Abstraction.

Reply:
yeah I did make note of the duplication. Waiting for the 3rd repetition I suppose: https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming) 😛